build_tools/buildkite/cmake/linux/x86_64/benchmark.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+steps:
+  - label: "Build"
```

Comment:
fyi you can use emojis in these labels to give them a bit more life and make scanning the steps easier

for example: https://buildkite.com/iree/iree-build-configurations/builds/2918 vs https://buildkite.com/iree/iree-benchmark/builds/2870

https://buildkite.com/docs/pipelines/emojis
https://github.com/buildkite/emojis

Reply:
Done.